### ISSUE TITLE
Add Cloth-working tools

### ISSUE THREAD

**Issue by VinterNacht** [wendall911/TinkerSurvival#10]
* Collect Cactus, process into 'Cactus Spines'
* Combine Spines with string and stick to make a 'wool comb'
* Use the wool comb to get 'Batt of Wool'
* Combine two stones to get a 'whorl'
* Combine a whorl with a stick to get a 'drop spindle'
* Combine Drop Spindle w/ Batt of Wool to get 'String' or 'Yarn'

Change Bed Recipe to require cloth instead of wool blocks (Do the same for other similar items)
Add Recipe for Loom, Require Loom to make cloth

**Comment by wendall911** [comment]
This is a really great idea. I can see a couple issues, and wonder if you have any ideas.
 First, requiring Cactus can be difficult depending on the seed. A biome with cactus could be 1000M away. The other issue is a bed. It is the only way to set a new spawn. Without the ability to easily get a bed, players end up clustering around the spawn, which ends up making adventuring problematic in the early game. One of the few uses for the knife currently is to convert wool blocks into string. That would need to go away, making the knife less useful.

If there are some solutions for balancing that could be come up with here, I'd love to incorporate this type of concept. Would extend the early game a bit more. Sounds fun :)

**Comment by VinterNacht** [comment]
Well, the easiest answer would be to just make wool-working tools be exactly like any other tool, give them tiers.  Historically wood was used for this sort of thing, though certainly not by preference. 

There's a whole thought pattern to be gone into here:





Knife -> Grass Fibers -> Weave Grass into grass mat.  Grass mats will set spawn but will not allow you to sleep. (Boom, problem solved for spawn reset).  Knife remains useful, even while not really gaining any new things it can do.

Primitive Fishing Rod -> Lower durability, made from stick and plant fiber.

Eat Fish - Use Knife w/ Fish -> Fish Bones

Fish Bones, Stick, Plant String -> Crude Wool Comb

Make the rest of the wool combs obtainable through the Tinker's Construct Tables.

Retain necessity for loom to make cloth.

 
If you're serious about adding this particular arc of materials/tech, let me know.  I can get much more in depth than this.

**Comment by VinterNacht** [comment]
Honestly, I don't think it falls within the scope of this mod, but I'd *love* to see just about everything require the Tinker's crafting system to make, incorporating new sub-combines.

Like take the vanilla chest recipe, remove it, and make chests take ingredients that increase their capacity.  Say, for instance:

Chests:
"Planks" -> Modify base number of stacks a chest can hold.
"Legs" -> Increase number of stacks a chest can hold.
"Hinges" -> Set Base size of each stack 
"Reinforcement" -> Increase size of possible stack 

In short: I'd love to see Tinker's Construct be HOW you craft... Period.

**Comment by wendall911** [comment]
Ok, this is excellent feedback. I'm not sure if it fits either, but really want to explore this either through additions to this mod, or another mod that extends with these ideas. I am out of town and not working on anything for the next week, but will get back to this and figure out what to do very soon. I do agree that the way crafting is done in Tinkers' Construct is far superior for crafting. It is fun really.

**Comment by VinterNacht** [comment]
No problem! When it comes to early game stuff I've got a pretty robust set of ideas.

**Comment by VinterNacht** [comment]
Ground control to Major Tom
